src/lib/api/index.ts

```diff
@@ -117,6 +119,13 @@ export async function post(agent: BskyAgent, opts: PostOpts) {
       const {width, height} = image.compressed || image
       logger.debug(`Uploading image`)
       const res = await uploadBlob(agent, path, 'image/jpeg')
+
+      if (isNative) {
+        try {
+          deleteAsync(path)
```

Comment:
If this throws asynchronously, does that crash the app or not? E.g. imagine its implementation is

```js
async function deleteAsync() {
  await new Promise(resolve => setTimeout(resolve, 1000))
  throw Error('boom')
}
```

If this doesn't lead to a redbox, we're good. If it does, I think we need to handle Promise rejection too.

Reply:
No red box. I suppose we could actually remove the try/catch completely actually. That being said, I think we should just err on the side of caution in all cases. Updating.